gooddata-sdk/tests/catalog/test_catalog_user_service.py

```diff
@@ -68,17 +71,18 @@ def test_update_user(test_config):
     user_id = test_config["test_user"]
     user = sdk.catalog_user.get_user(user_id)
     new_auth_id = f"{user_id}_123"
-    new_user_groups = ["demoGroup", "visitorsGroup"]
+    user_group_ids = ["demoGroup", "visitorsGroup"]
     try:
-        sdk.catalog_user.create_or_update_user(user_id, new_auth_id, new_user_groups)
+        user_e = CatalogUser.create_user(user_id=user_id, authentication_id=new_auth_id, user_group_ids=user_group_ids)
```

Comment:
Confusing method name, first I thought it calls the backend.
Please, rename all similar methods to `init`.